main.py

```diff
@@ -20,12 +20,6 @@ def main(config):
     random.seed(None)
     # Set other random seeds
     set_seeds(config.seed)
-    # Log config
-    # TODO: Move log config to Logger
-    log_config = flatten_config(OmegaConf.to_container(config, resolve=True), sep="/")
```

Comment:
Best to remove unused import `from gflownet.utils.common import flatten_config`.